staging/src/k8s.io/api/node/v1beta1/types.go

```diff
@@ -65,6 +72,29 @@ type Overhead struct {
 	PodFixed corev1.ResourceList `json:"podFixed,omitempty" protobuf:"bytes,1,opt,name=podFixed,casttype=k8s.io/api/core/v1.ResourceList,castkey=k8s.io/api/core/v1.ResourceName,castvalue=k8s.io/apimachinery/pkg/api/resource.Quantity"`
 }
 
+// Scheduling specifies the scheduling constraints for nodes supporting a
+// RuntimeClass.
+type Scheduling struct {
+	// nodeSelector lists labels that must be present on nodes that support this
+	// RuntimeClass. Pods using this RuntimeClass can only be scheduled to a
+	// node matched by this selector. The RuntimeClass nodeSelector is merged
+	// with a pod's existing nodeSelector. Any conflicts will cause the pod to
+	// be rejected in admission.
+	// +optional
+	NodeSelector map[string]string `json:"nodeSelector,omitempty" protobuf:"bytes,1,opt,name=nodeSelector"`
+
+	// tolerations are appended (excluding duplicates) to pods running with this
+	// RuntimeClass during admission, effectively unioning the set of nodes
+	// tolerated by the pod and the RuntimeClass.
+	// +optional
+	// +listType=map
```

Comment:
Do we really want this to be patchable?  How well does machinery handle the compound keys?  ISTR @apelisse advising me not to do this if we could avoid it...

IOW - should this be "atomic" instead?

Reply:
sync'd w/ @apelisse offline, he convinced me to go with atomic